chromadb/utils/query_helper.py

```diff
@@ -0,0 +1,135 @@
+from typing import List, Union, Literal, Any, Dict, cast
+
+from chromadb.types import (
+    WhereOperator,
+    LiteralValue,
+    WhereDocument as WhereDocumentType,
```

Comment:
I don't love that we stomp on our own type names. Maybe lets call these Builders?

WhereBuilder()? Or something. I don't love that either. CC @jeffchuber for ideas.